spec/integration/api/v2010/account/call_spec.rb

```diff
@@ -71,6 +71,51 @@
     expect(actual).to_not eq(nil)
   end
 
+  it "receives create_with_twiml responses" do
```

Comment:
Don't think these changes should be here.

Reply:
Would this be because I did not pull in the latest changes from the `master` branch?

Reply:
More likely you're working off the latest api-defs and not what was last used for generation.